src/libcollections/vec.rs

```diff
@@ -476,6 +476,24 @@ impl<T> Vec<T> {
     /// Note that this will drop any excess capacity. Calling this and
     /// converting back to a vector with `into_vec()` is equivalent to calling
     /// `shrink_to_fit()`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// let v = vec!(1, 2, 3);
```

Comment:
please use the `[]` style rather than the `()` style

and put a newline between these two things